pipeline/frontend/yaml/linter/schema/schema.json

```diff
@@ -14,118 +14,42 @@
     "variables": {
       "description": "Use yaml aliases to define variables. Read more: https://woodpecker-ci.org/docs/usage/advanced-yaml-syntax"
     },
-    "clone": {
-      "$ref": "#/definitions/clone"
-    },
-    "skip_clone": {
-      "type": "boolean"
-    },
-    "branches": {
-      "$ref": "#/definitions/branches"
-    },
-    "when": {
-      "$ref": "#/definitions/pipeline_when"
-    },
-    "steps": {
-      "$ref": "#/definitions/step_list"
-    },
-    "pipeline": {
-      "$ref": "#/definitions/step_list",
-      "description": "deprecated, use steps"
-    },
-    "services": {
-      "$ref": "#/definitions/services"
-    },
-    "workspace": {
-      "$ref": "#/definitions/workspace"
-    },
-    "matrix": {
-      "$ref": "#/definitions/matrix"
-    },
-    "platform": {
-      "$ref": "#/definitions/platform"
-    },
-    "labels": {
-      "$ref": "#/definitions/labels"
-    },
+    "clone": { "$ref": "#/definitions/clone" },
+    "skip_clone": { "type": "boolean" },
+    "branches": { "$ref": "#/definitions/branches" },
+    "when": { "$ref": "#/definitions/pipeline_when" },
+    "steps": { "$ref": "#/definitions/step_list" },
+    "pipeline": { "$ref": "#/definitions/step_list", "description": "deprecated, use steps" },
+    "services": { "$ref": "#/definitions/services" },
+    "workspace": { "$ref": "#/definitions/workspace" },
+    "matrix": { "$ref": "#/definitions/matrix" },
+    "platform": { "$ref": "#/definitions/platform" },
+    "labels": { "$ref": "#/definitions/labels" },
     "depends_on": {
       "type": "array",
       "minLength": 1,
-      "items": {
-        "type": "string"
-      }
+      "items": { "type": "string" }
     },
     "runs_on": {
       "type": "array",
       "minLength": 1,
-      "items": {
-        "type": "string"
-      }
-    },
-    "version": {
```

Comment:
We might leave it in here as it will throw an error otherwise if someone added the version property.